py/sentry_relay/processing.py

```diff
@@ -261,6 +262,26 @@ def validate_project_config(config, strict: bool):
         raise ValueError(error)
 
 
+def normalize_global_config(config):
+    """Normalize the global config.
+
+    Normalization consists on deserializing and serializing back the given
+    global config. If deserializing fails, throw an exception. Note that even if
+    the roundtrip doesn't produce errors, the given config may differ from
+    normalized one.
+
+    :param config: the global config to validate.
+    """
+    assert isinstance(config, str)
+    rv = rustcall(lib.normalize_global_config, encode_str(config))
+    try:
+        decoded = decode_str(rv, free=True)
+        parsed = json.loads(decoded)
+        return parsed
+    except Exception:
+        raise ValueError("Normalization error: incorrect input config")
```

Comment:
I would remove this `try` `except` block and just let the original exception bubble up to expose full information (for example, JSON parsing errors).